__tests__/integration-metrics.test.js

```diff
@@ -165,7 +151,7 @@ describe('Metrics SDK Integration Tests', function () {
       email: 'owlbert@example.com',
     });
 
-    expect(har.clientIPAddress).to.equal('127.0.0.1');
+    expect(har.clientIPAddress).to.match(/\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}/);
```

Comment:
`clientIPAddress` is now the IP of the Docker container so we needed to loosen this assertion up a bit. No this doesn't support IPv6 but we also aren't sending IPv6 addresses back to Metrics right now in any SDK.